README.md

```diff
@@ -1,23 +1,29 @@
-# Appcelerator Titanium Package for Atom [![Build Status](https://travis-ci.org/appcelerator/atom-appcelerator-titanium.svg?branch=master)](https://travis-ci.org/appcelerator/atom-appcelerator-titanium)
+# Titanium SDK Package for Pulsar [![Build Status](https://travis-ci.org/appcelerator/atom-appcelerator-titanium.svg?branch=master)](https://travis-ci.org/appcelerator/atom-appcelerator-titanium)
 
-Appcelerator Titanium build tools and UI for the [Atom text editor](https://atom.io).
+Titanium SDK build tools and UI for the [Pulsar text editor](https://pulsar-edit.dev/) (formally Atom).
 
-<img src="https://user-images.githubusercontent.com/10667698/42400977-8e552cde-8141-11e8-8696-1380eb9dd581.png" width="900px" alt="Appcelerator Titanium for Atom screenshot" />
+<img src="https://user-images.githubusercontent.com/10667698/42400977-8e552cde-8141-11e8-8696-1380eb9dd581.png" width="900px" alt="Titanium SDK for Pulsar screenshot" />
 
 ## Getting Started
 
-* Atom 1.21 (or later) is required.
-* Some features are implemented as providers for services in the [Atom IDE](https://ide.atom.io) package. Install `atom-ide-ui` package.
-* Install `appcelerator-titanium` package.
+* Pulsar is required.
+* Install `pulsar-titanium` package.
```

Comment:
Package name is just `titanium`, I think we do still need some extra packages but I'll need to check what else is needed

```suggestion
* Install `titanium` package.
```